### PR TITLE
Issue 61 completed

### PR DESCRIPTION
**Pull request by remingtontr**
Article for SDIO/eMMC rewritten.

### CODE REVIEW

File: docs/hardware-interfaces/sdio.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  
+  The SDIO/eMMC pins are highlighted on the Omega2S diagram below.
+  
+
+![omega2s-sdio_emmc-diagram](./assets/omega2s-sdio_emmc-pinout.png)
+
+  </TabItem>
+</Tabs>
+
+See [Omega2S hardware design guide](https://github.com/OnionIoT/Omega2/blob/master/Documents/Omega2S%20Hardware%20Design%20Guide.pdf) for more information and recommendations on designing boards with microSD card slots or eMMC.
```
Comment (greenbreakfast) [inline docs/hardware-interfaces/sdio.md:45]: What do you think about folding this into the Omega2S tab above? 
It only applies to the omega2s/2s+, boards that use the Omega2+ must use the microSD slot on the Omega2+ itself.

**Review by greenbreakfast — COMMENTED** [review]
Added 2 formatting suggestions, please take a look

File: docs/hardware-interfaces/sdio.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# SDIO / eMMC
+
+## Introduction
+The Omega2 features an SDIO/eMMC interface that enables the use of SD/MicroSD cards or eMMC storage chips.
+
+**Omega2:** Does not have a microSD slot and does not expose the SDIO/eMMC interface.
```
Comment (greenbreakfast) [inline docs/hardware-interfaces/sdio.md:13]: Let's have this in a bullet-point list so it renders a little more cleanly

**Comment by remingtontr** [comment]
Yep, can do that.

Tom Remington
________________________________

**Comment by remingtontr** [comment]
Requested changes made and pushed.

Tom Remington
________________________________

**Comment by greenbreakfast** [comment]
Made a small adjustment to the bullet point list to minimize white-space
Merged in

cc #61

**Pull request merged**